tutorials/tour/_posts/2017-02-13-basics.md

```diff
@@ -170,6 +185,7 @@ class Greeter(prefix: String, suffix: String) {
     println(prefix + name + suffix)
 }
 ```
+The return type of the method `greet` is `Unit` which says there's nothing meaningful to return. It's used similarly to `void` in Java and C but the main difference is that there is actually a singleton value of type `Unit`, written `()`. 
```

Comment:
perhaps:

The return type of the method `greet` is `Unit`, which says there's nothing meaningful to return. It's used similarly to `void` in Java and C. (A difference is that because every Scala expression must have _some_ value, there is actually a singleton value of type `Unit`, written `()`. It carries no information.)